index.js

```diff
@@ -37,12 +37,14 @@ module.exports = function headless(options, startnum, callback) {
   }
 
   findFreeServernum(startnum, function(servernum) {
-    if (!options) {
-      var childProcess = spawn('Xvfb', [':' + servernum]);
-    }
-    else {
-      var childProcess = spawn('Xvfb', [':' + servernum , '-screen' , '0' , options.display.width + 'x' + options.display.height + 'x' + (options.display.depth || 16) ]);
+    options = options || {}
+
+    var args = options.args || []
+    var display = []
+    if (options.display) {
+      display = ['-screen', '0', options.display.width + 'x' + options.display.height + 'x' + (options.display.depth || 16)]
     }
+    var childProcess = spawn('Xvfb', [':' + servernum].concat(display).concat(args));
```

Comment:
All right, I added an `args` variable as you suggested. I left `display` as multiple lines, since it's really long as one line. Does that work?